pkgs/development/interpreters/python/wrapper.nix

```diff
@@ -35,14 +35,22 @@ let
       fi
       mkdir -p "$out/bin"
 
+      rm -f $out/bin/.*-wrapped
+
       for path in ${lib.concatStringsSep " " paths}; do
         if [ -d "$path/bin" ]; then
           cd "$path/bin"
           for prg in *; do
             if [ -f "$prg" ]; then
               rm -f "$out/bin/$prg"
               if [ -x "$prg" ]; then
-                makeWrapper "$path/bin/$prg" "$out/bin/$prg" --set NIX_PYTHONPREFIX "$out" --set NIX_PYTHONEXECUTABLE ${pythonExecutable} --set NIX_PYTHONPATH ${pythonPath} ${lib.optionalString (!permitUserSite) ''--set PYTHONNOUSERSITE "true"''} ${lib.concatStringsSep " " makeWrapperArgs}
+                if [ -f ".$prg-wrapped" ]; then
+                  echo "#!${pythonExecutable}" > "$out/bin/$prg"
```

Comment:
Correct me if I am wrong, but does this assume that *all* executables in linked Python packages are Python scripts?

Reply:
No, it only assumes that if files named `bin/foo` _and_ `bin/.foo-wrapped` exist, then `bin/foo` is a wrapper and `bin/.foo-wrapped` is a python script.

Reply:
> In time we should aim to not wrap packages at build time.

Yes! I have an idea for how to do that.